invenio_rdm_records/cli.py

```diff
@@ -116,9 +116,6 @@ def create_fake_record():
         "contributors": [{
             "name": fake.name(),
             "type": "Personal",
-            "identifiers": {
```

Comment:
doubt: what is wrong with this (appart from the value which might not pass the regex?)

Reply:
Indeed that Orcid would not pass the regex. We allow no identifiers for a contributor/creator, so it is good to show this on the generated record. That's mostly it. Do you have a replacement in mind?

Reply:
Not really, was just nice to display the orcid badge on the contributors, etc. on the demo landing page. But it's okay it will come eventually when we have real world demo records from partners

Reply:
Yes and the Orcid badge appears for creators at least so they will be able to see it in action a little.